gitea/files/options/locale/locale_nb-NO.ini

```
@@ -56,101 +78,277 @@ forks = Forks
 activities = Aktiviteter
 pull_requests = Pull requests
 issues = Issues
+milestones = Milepæler
 
+ok = OK
 cancel = Avbryt
+retry = Retry
+rerun = Re-run
+rerun_all = Re-run all jobs
+save = Lagre
+add = Legg til
+add_all = Legg til alle
+remove = Fjern
+remove_all = Fjern alle
+remove_label_str = Fjern element "%s"
+edit = Rediger
+view = View
+
+enabled = Aktivert
+disabled = Deaktivert
+locked = Locked
+
+copy = Kopier
+copy_url = Kopier URL
+copy_hash = Copy hash
+copy_content = Kopier innhold
+copy_branch = Kopier navn på gren
+copy_success = Kopiert!
+copy_error = Kopiering feilet
+copy_type_unsupported = Denne filtypen kan ikke kopieres
+
+write = Skriv
+preview = Forhåndsvis
+loading = Laster…
+
+error = Feil
+error404 = Siden du prøver å nå <strong>finnes ikke</strong> eller så <strong>er du ikke autorisert</strong> for å vise den.
+go_back = Go Back
+invalid_data = Invalid data: %v
+
+never = Aldri
+unknown = Ukjent
+
+rss_feed = RSS-strøm
+
+pin = Pin
+unpin = Unpin
+
+artifacts = Artifacts
+confirm_delete_artifact = Are you sure you want to delete the artifact '%s' ?
```

confirm_delete_artifact = Er du sikker på at du vil slette artefakten '%s'?
```suggestion
confirm_delete_artifact = Are you sure you want to delete the artifact '%s' ?
```

---

Jeg fant akkurat "suggestion" @mlqn - så da gjør jeg det, så jobben din blir lettere :-).